### ISSUE TITLE
idea: facebook DU group administration integration

### ISSUE THREAD

**Issue by compwron** [doubleunion/arooo#450]
Allow an arooo admin to add/remove facebook Admin/Editor role from DU members via a button in the UI

**Comment by compwron** [comment]
https://developers.facebook.com/docs/groups-api/
https://developers.facebook.com/docs/pages/managing/
https://developers.facebook.com/docs/graph-api/reference/page/

**Comment by brittag** [comment]
I don't think we have any active Facebook groups for DU - I believe people tried making a couple of them, but it never went anywhere. I'd be in favor of us deleting the Facebook groups instead of reviving them.

We do have a Facebook page though (https://www.facebook.com/doubleunion), which is useful for making Facebook events sometimes. It could still be nice to have the ability to view/add/remove roles for the page from the internal webapp.
